platform/fabric/core/generic/config/config.go

```diff
@@ -95,13 +95,14 @@ func (c *Config) TLSClientCertFile() string {
 }
 
 func (c *Config) KeepAliveClientInterval() time.Duration {
-	return c.configService.GetDuration("fabric." + c.prefix + "tls.keepalive.client.interval")
+	return c.configService.GetDuration("fabric." + c.prefix + "keepalive.interval")
 }
 
 func (c *Config) KeepAliveClientTimeout() time.Duration {
-	return c.configService.GetDuration("fabric." + c.prefix + "tls.keepalive.client.timeout")
+	return c.configService.GetDuration("fabric." + c.prefix + "keepalive.timeout")
 }
 
+/*
```

Comment:
let's remove this

Reply:
oops thought I had done that, will remove